src/main/scala/com/datastax/gatling/plugin/model/DseCqlStatementBuilders.scala

```diff
@@ -6,7 +6,15 @@
 
 package com.datastax.gatling.plugin.model
 
-import com.datastax.driver.core.{PreparedStatement, SimpleStatement}
+import com.datastax.oss.driver.api.core.cql.{
+  BatchStatement => BatchS,
```

Comment:
Open question: would it help to name driver types with a common prefix/suffix to make them more easily distinguishable? E.g. `DriverBatchStmt`, `DriverBatchBuilder`...

Reply:
Personally I don't think so; I think the names as we have them are distinct enough while still having the advantage of brevity.

Reply:
Why even alias them? They don't clash with any project types.

This makes the code harder to read. Granted, it's not a big deal if you read the file top to bottom, but consider other common situations: jumping to a method definition from another file, reading a diff that might not include the imports, etc.

Brevity doesn't matter, you can use IDE completion.

Reply:
I disagree; I'd argue brevity does matter for reading comprehension.  The repetition of "Statement" here is just noise.

Reply:
It's not noise, it's part of the name that precisely identifies the driver type. A name that anyone from the team instantly recognizes, even if they've not worked on the gatling plugin before.

Yes, I can read the code with the aliases, but "SimpleStatementBuilder => SimpleB" is just another piece of information that I have to keep in my head. If I forgot, I have to go check the imports (hopefully I'm reading the full file, not a diff), or maybe my IDE can show me a tooltip (hopefully I am in my IDE, not on Github or patching something with vi on a CI box).

What if someone else uses their own aliases in another project (or even different parts of the same project)? `BoS` and `BoB` are pretty short too. Or maybe they use `SimpleB` for something else (e.g. `javax.script.SimpleBindings`). Imagine if you don't have an IDE available and you need to grep the codebase for every place where `SimpleStatementBuilder` is referenced...

Renaming clauses have their use when the type you want to import clashes with something that you already have in scope:
```scala
// Assuming we also intend to use scala.collection.immutable.Map
import java.util.{Map => JavaMap}
```

That's not the case here, so why introduce an unnecessary level of indirection?

Reply:
I agree with @olim7t , the `Statement` part of the type name doesn't seem to be a big problem - the name is not very long and we are using those types everywhere.

Reply:
I completely disagree, but in the spirit of trying to get this wrapped up I'll make the change.  For me this is nearly identical to imports of static methods which can go _a long_ way towards reducing the verbiage in a given class.  The question of viewing this code in a diff feels like something of a canard to me as diffs _always_ have limited context by their very nature.